docs/rest_api.rst

```diff
@@ -544,7 +543,7 @@ be rejected.
 
 .. http:get:: /api/(version)/events/network
 
-   Query for registry network events.
+   Query for token network creations.
```

Comment:
This will get all events of the registry contract at ``registry_address``, and since the [TokenNetworkRegistry](https://github.com/raiden-network/raiden-contracts/blob/master/raiden_contracts/contracts/TokenNetworkRegistry.sol#L22) returns only ``event TokenNetworkCreated(address indexed token_address, address indexed token_network_address);``. I don't think it's wrong.